pkg/crc/constants/constants_darwin.go

```diff
@@ -9,7 +9,7 @@ import (
 const (
 	OcExecutableName           = "oc"
 	PodmanRemoteExecutableName = "podman"
-	TrayExecutableName         = "CodeReady Containers.app"
+	TrayExecutableName         = "CRC.app"
```

Comment:
@anjannath @gbraad I think we go with `crc.app` here?

Reply:
This should be `OpenShift Local.app`